static/app/views/automations/components/automationStatsChart.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from '@emotion/styled';
+
+import {BarChart} from 'sentry/components/charts/barChart';
+import ChartZoom from 'sentry/components/charts/chartZoom';
+import {HeaderTitleLegend} from 'sentry/components/charts/styles';
+import type {DateTimeObject} from 'sentry/components/charts/utils';
+import Panel from 'sentry/components/panels/panel';
+import PanelBody from 'sentry/components/panels/panelBody';
+import PanelFooter from 'sentry/components/panels/panelFooter';
+import Placeholder from 'sentry/components/placeholder';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Automation, AutomationStats} from 'sentry/types/workflowEngine/automations';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import RouteError from 'sentry/views/routeError';
+
+interface IssueAlertDetailsProps extends DateTimeObject {
+  automationId: Automation['id'];
+}
+
+export function AutomationStatsChart({
+  automationId,
+  period,
+  start,
+  end,
+  utc,
+}: IssueAlertDetailsProps) {
+  const organization = useOrganization();
+  const {
+    data: fireHistory,
+    isPending,
+    isError,
+    error,
+  } = useApiQuery<AutomationStats[]>(
+    [
+      `/organizations/${organization.slug}/workflows/${automationId}/stats/`,
+      {
+        query: {
+          ...(period && {statsPeriod: period}),
+          start,
+          end,
+          utc,
+        },
+      },
+    ],
+    {staleTime: 30000}
+  );
+
+  const totalAlertsTriggered =
+    fireHistory?.reduce((acc, curr) => acc + curr.count, 0) ?? 0;
+
+  if (isError) {
+    return <RouteError error={error} />;
```

Comment:
RouteError is meant to be used on a full-page error so will look bad here. You probably want <LoadingError /> and to rendering it inside the panel (like you have for Placeholder)